README.md

```diff
@@ -69,17 +69,6 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-Start HorariumServer in Asp.NET Core application
```

Comment:
Maybe add comment above about that HorariumServer is registered as HostedService?

Reply:
Updated readme, added a note about hosted service